packages/file_selector/file_selector_macos/lib/src/messages.g.dart

```diff
@@ -0,0 +1,227 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
```

Comment:
Is there any reason we can't use the same gen file for all 3 locations?

Reply:
What do you mean "all three locations"?